FreeRTOS/FreeRTOS
# Add MQTT Server Auth (Basic TLS) demo

Description
-----------
Porting MQTT basic TLS demo from CSDK. 

Demo shows use of the managed MQTT API over a server-authenticated connection.

Test Steps
-----------
Configure demo as usual -- ensure democonfigROOT_CA_PEM in demo_config.h is defined to proceed with server authentication.


TBD: Adding and maintaining record of outgoing publish packets to allow for retransmission in mutual auth and server auth demos.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
